<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="push_button_ready">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Continuar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_instructions">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Seleccione el archivo excel a utilizar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_file_selector">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Abrir archivo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_path_info">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>No hay archivo seleccionado</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_path">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
